catalog/github/webhook.js

```diff
@@ -88,6 +88,66 @@ function main(params) {
     });
 
     return whisk.async();
+  } else if(lifecycleEvent === 'DELETE') {
+    //list all the existing webhooks first.
+    var options = {
+        method: 'GET',
+        url: registrationEndpoint,
+        headers: {
+            'Content-Type': 'application/json',
+            'Authorization': authorizationHeader,
+            'User-Agent', 'whisk'
```

Comment:
@lzbj Syntax error. Should be a colon instead of a comma. Please ensure that your code runs as expected before putting up for review.